src/vivarium/framework/lookup/manager.py

```diff
@@ -64,9 +64,9 @@ def setup(self, builder: "Builder") -> None:
     def build_table(
         self,
         data: LookupTableData,
-        key_columns: Union[List[str], Tuple[str, ...]],
-        parameter_columns: Union[List[str], Tuple[str, ...]],
-        value_columns: Union[List[str], Tuple[str, ...]],
+        key_columns: Sequence[str],
```

Comment:
I'm confused how this is a Sequence if you originally defined it as [] but I haven't really used Sequence.

Reply:
ultimately it can be any sequence (i.e. list or tuple). But it's kinda weird b/c as you said we instantiate it specifically as a list. But we were _already_ saying `Union[list, tuple]` anyway! So I dunno - I think we're just a little inconsistent here in general